source/classes.tex

```diff
@@ -4154,7 +4154,7 @@
 };
 
 void f() {
-  A::BB x;          // OK, typedef name \tcode{A::BB} is public
+  A::BB x;          // OK, \grammarterm{typedef-name} \tcode{A::BB} is public
```

Comment:
```suggestion
  A::BB x;          // OK, typedef \tcode{A::BB} is public
```
(because access is a property of the member, not the name).

Reply:
fixed